abci/example/kvstore/persistent_kvstore.go

```diff
@@ -45,34 +42,3 @@ func (app *PersistentKVStoreApplication) OfferSnapshot(req types.RequestOfferSna
 func (app *PersistentKVStoreApplication) ApplySnapshotChunk(req types.RequestApplySnapshotChunk) types.ResponseApplySnapshotChunk {
 	return types.ResponseApplySnapshotChunk{Result: types.ResponseApplySnapshotChunk_ABORT}
 }
-
-func (app *PersistentKVStoreApplication) ExtendVote(req types.RequestExtendVote) types.ResponseExtendVote {
-	return types.ResponseExtendVote{VoteExtension: ConstructVoteExtension(req.Vote.ValidatorAddress)}
-}
-
-func (app *PersistentKVStoreApplication) VerifyVoteExtension(req types.RequestVerifyVoteExtension) types.ResponseVerifyVoteExtension {
-	return types.RespondVerifyVoteExtension(app.verifyExtension(req.Vote.ValidatorAddress, req.Vote.VoteExtension))
-}
```

Comment:
Nothing seems to be automatically using/testing for this behavior. Perhaps in a follow-up PR we could implement something that could be added to the E2E tests?

Reply:
I thought that the default behavior (from the embedded `BaseApplication`) would apply here in the absence of overrides? The default behavior doesn't extend any votes, and just accepts any vote extensions.

Reply:
Ugh. That embedding hack. Right, that should cover it.